FunctionalTableData/CellStyle.swift

```diff
@@ -73,14 +88,8 @@ public struct CellStyle {
 	}
 	
 	func configure(cell: UICollectionViewCell, in collectionView: UICollectionView) {
-		if let backgroundView = backgroundView {
-			cell.backgroundView = backgroundView
-		} else {
-			cell.backgroundColor = backgroundColor ?? UIColor.white
-			let backgroundView = UIView()
-			backgroundView.backgroundColor = cell.backgroundColor
-			cell.backgroundView = backgroundView
-		}
+		cell.backgroundColor = backgroundColor
+		cell.backgroundView = backgroundView
```

Comment:
UITableViewCell.backgroundView is nil in vanilla UIKit by default right? We're not nilling out something that had a value now?

Reply:
Did some testing with a tableview created in code in a new project:

- `UITableViewCell.backgroundView` is nil by default and remains nil when the cell appears.
- `UITableViewCell.backgroundColor` is "nil" by default, but gets set to `UIColor.white` at some point between `tableView(_:cellForRowAt:)` and when the cell appears.
- ...*except* if you set the color manually to `nil` in `tableView(_:cellForRowAt:)`, then it remains `nil` when the cell appears.

So giving `CellStyle` a default background color of white (but letting you override it to nil) mimics how UIKit works out of the box.